### PR TITLE
Add troubleshoot section about SSL certificate generation issue

### PR DESCRIPTION
**Pull request by mzottola**

### CODE REVIEW

File: website/docs/using-qovery/troubleshoot.md [hunk 1]
```diff
@@ -168,6 +168,35 @@ Open a ticket to the Cloud Provider support, and as to raise this limit.
 </ol>
 </Steps>
 
+## SSL Certificate expired for a custom domain
```
Comment (mzottola) [inline website/docs/using-qovery/troubleshoot.md:171]: Can't get my SSL / TLS Certificate

File: website/docs/using-qovery/troubleshoot.md [hunk 2]
```diff
@@ -168,6 +168,35 @@ Open a ticket to the Cloud Provider support, and as to raise this limit.
...
+
+When a custom domain is added to an application, it must be configured on your side according the instructions displayed:
+
+<p Valign="center">
+  <img src="/img/custom-domain-configuration.png" alt="Custom Domain Configuration" />
+</p>
+
+Internally we are using [cert-manager](https://cert-manager.io/docs/) to handle SSL certificates generation.
```
Comment (mzottola) [inline website/docs/using-qovery/troubleshoot.md:179]: this line to remove

File: website/docs/using-qovery/troubleshoot.md [hunk 3]
```diff
@@ -168,6 +168,35 @@ Open a ticket to the Cloud Provider support, and as to raise this limit.
...
 </Steps>
 
+## SSL Certificate expired for a custom domain
+
+When a custom domain is added to an application, it must be configured on your side according the instructions displayed:
+
+<p Valign="center">
+  <img src="/img/custom-domain-configuration.png" alt="Custom Domain Configuration" />
+</p>
+
```
Comment (mzottola) [inline website/docs/using-qovery/troubleshoot.md:178]: add smthing like "check with dig command with + short)

File: website/docs/using-qovery/troubleshoot.md [hunk 4]
```diff
@@ -168,6 +168,49 @@ Open a ticket to the Cloud Provider support, and as to raise this limit.
...
+You can check that your custom domain is well configured by executing the following command:
+```
+dig A www.your-custom-domain.com +short
+```
+You should retrieve in the `ANSWER SECTION` the default url configured by Qovery: zxxxxxxxx-zxxxxxxxx-gtw.zxxxxxxxx.qovery.cloud.
```
[the remark this answers is not in the document]
Comment (deimosfr) [reply website/docs/using-qovery/troubleshoot.md:183]: if possible, use the one from the screenshot

[the remark this answers is not in the document]
Comment (mzottola) [reply website/docs/using-qovery/troubleshoot.md:183]: Is it okay if I use the new qovery console domain ? It would be a real example

**Pull request merged**